honeybadgermpc/mpc.py

```diff
@@ -239,6 +239,23 @@ def __mul__(self, other):
 
         def __str__(self): return '{%d}' % (self.v)
 
+        def __div__(self, other):
+            if MixinOpName.InvertShare not in context.config:
```

Comment:
I am wondering if we should write a MixinValidator, now that we have started creating dependent mixins this might end up with a lot of if, else checks. What do you think @Drake-Eidukas ?

Reply:
I think a validator might not be too impossible-- especially if we are considering redoing some of the way we do mixins (i.e. moving stuff into progs). Could be interesting to create some way to have the mixins define the operations they need. I'll think about this some more when I start tackling the refactors to mixins